package.json

```diff
@@ -31,11 +31,11 @@
     "cosmiconfig": "^5.0.5"
   },
   "devDependencies": {
+    "@babel/core": "^7.1.0",
     "ast-pretty-print": "^2.0.1",
-    "babel-core": "7.0.0-beta.3",
     "babel-plugin-tester": "^5.0.0",
     "babel-types": "^6.26.0",
-    "babylon": "7.0.0-beta.34",
+    "babylon": "^6.18.0",
```

Comment:
Any chance we could bump this to 7.0.0? I'd prefer to leave babel 6.x behind for everything.

Reply:
Good point!